### PR TITLE
Bring back save settings option for exporter

### PR DESCRIPTION
**Pull request by pjoe**
- Use suggested name and description from #109

### CODE REVIEW

File: addons/io_scene_gltf2/__init__.py [hunk 1]
```diff
@@ -255,7 +255,10 @@ class ExportGLTF2_Base:
         default=False
     )
 
-    will_save_settings = BoolProperty(default=False)
+    will_save_settings = BoolProperty(
+        name='Remember Settings',
+        description='Store glTF export settings in the Blender project',
```
Comment (donmccurdy) [inline addons/io_scene_gltf2/__init__.py]: maybe, `Remember Export Settings`? /cc @emackey

**Comment by donmccurdy** [comment]
I don't have a strong preference on this... it hasn't been something I've needed, but I can see where others might. I have some reservations about writing extra data into the Blender project when other exporters don't, as it has caused issues in the past with "breaking" a file if exporter settings change.

**Comment by emackey** [comment]
> maybe, `Remember Export Settings`

I'd be fine with the name changing to that.  I think the description is good as-is.

> it has caused issues in the past with "breaking" a file if exporter settings change

I just tested that, it drops the settings but doesn't crash.  At some point I think we added a try/catch or something to protect against those crashes.

File: addons/io_scene_gltf2/__init__.py (hunk 1 above)
Comment (pjoe) [reply addons/io_scene_gltf2/__init__.py]: Fixed

**Comment by emackey** [comment]
Any objections?  @julienduroure @UX3D-nopper

**Comment by pjoe** [comment]
I added a quick explanation in io_gltf2.rst

**Pull request merged**